transform/snowflake-dbt/models/hightouch/freemium/enterprise_trial_requests/etr_campaignmember_insert.sql

```diff
@@ -0,0 +1,54 @@
+{{config({
+    "schema": "hightouch",
+    "materialized": "view",
+    "tags":["hourly","freemium"]
+  })
+}}
+with existing_members as (
+    select
+        campaignmember.sfid,
+        campaignmember.email,
+        campaignmember.dwh_external_id__c,
+        row_number() over (partition by campaignmember.email order by createddate desc) as row_num
+    from {{ ref('campaignmember') }}
+    where campaignmember.campaignid = '7016u0000002OHjAAM'
+    -- where campaignmember.campaignid = '7013p000001U2GbAAK'
+), existing_leads as (
+    select
+        lead.sfid,
+        lead.email,
+        lead.dwh_external_id__c,
+        row_number() over (partition by lead.email order by createddate desc) as row_num
+    from {{ ref('lead') }}
```

Comment:
in another file, you've renamed `lead` to `freemium_leads` - should we be consistent naming-wise across these files?